sys-boot/grub/files/2.02_beta3-grub-mkconfig-btrfs-multidev.patch

```diff
@@ -0,0 +1,11 @@
+--- a/util/grub-mkconfig.in	2016-02-14 11:50:41.000000000 +0100
```

Comment:
Please add a proper patch header. This should include an explanation of the changes being made and a link to the bug report.

Ideally, make your changes against a copy of the upstream git repository, commit it on a branch, and create a patch using `git format-patch`. This patch can/should then be sent upstream as well.

At a minimum, please remove the useless time-stamps.